### DISCUSSION TITLE
After GitHub Desktop installation my commit's author changed

### DISCUSSION THREAD

**Discussion by alpceylan** [community/community#24093] [Apps, API and Webhooks]
<p>I installed Github Desktop to my computer and then author of my commits changed. It was like this: <a href="https://ibb.co/j4S3qRX" rel="noopener nofollow ugc">https://ibb.co/j4S3qRX</a><br>
And became this:<br>
<a href="https://ibb.co/7pBbcbr" rel="noopener nofollow ugc">https://ibb.co/7pBbcbr</a></p>
<p>How can I fix it?</p>

**Comment by airtower-luna** [comment]
<p>Github uses the commit mail address to match your commits with your account. To make your username and avatar appear again you need to set the commit email address on your system, or, if it’s already correct, add that mail address to your Github account.<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address" target="_blank" rel="noopener">Setting your commit email address - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>I’m not familiar with Github Desktop, so I don’t know why installing it would change the setting. Maybe someone else can answer that?</p>

**Comment by steveward** [comment, marked as answer]
<p><a class="mention" href="/alpceylan">@alpceylan</a> GitHub Desktop uses the name and email found in your <code>.gitconfig</code>. You can change this by going to the menu in GitHub Desktop and selecting <code>File</code> &gt; <code>Options</code> &gt; <code>Git</code>. Installing the app should not change this unless your name and email were not previously set in your <code>.gitconfig</code>, and in that case GitHub Desktop autofills with the name and email address from your GitHub account. I hope that helps!</p>

**Comment by alpceylan** [comment]
<p>I fixed it thank you Steve</p>
